### PR TITLE
Wordsmithing, edits, added landing cards

### PR DESCRIPTION
**Pull request by jrhoun**
Please take a look at my changes to make sure they're kosher. One weakness i haven't fixed is that the "final" sections (`Upgrade Basics`, `Upgrade Stability and Performance`, etc) don't have READMEs in there so we get blank pages when people go into them:

image

Two options are either a. add landing cards to each section - where each card = an article (similar to getting started section), or b. we add the readmes in so there's at least a TOC of sorts there.

### CODE REVIEW

**Comment by jhinkey** [comment]
Started reviewing. Thanks.

File: docs/dxp/7.x/en/installation-and-upgrades/upgrading-liferay-dxp/upgrade-stability-and-performance/database-tuning-for-upgrades.md [hunk 1]
```diff
@@ -46,4 +46,6 @@ The default configuration works well. It configures [asynchronous I/O to disk](h
 
 Turn off [synchronous commits](https://www.postgresql.org/docs/10/wal-async-commit.html) and set the [write ahead log writer delay](https://www.postgresql.org/docs/10/wal-async-commit.html) to `1000` milliseconds.
 
-Configurations on tuning your database for upgrading. When you're done upgrading, make sure tune your database for production again.
\ No newline at end of file
+## Conclusion
+
+When you're done upgrading, make sure revert your database configurations back to their production settings.
```
Comment (jhinkey) [inline docs/dxp/7.x/en/installation-and-upgrades/upgrading-liferay-dxp/upgrade-stability-and-performance/database-tuning-for-upgrades.md]: Typo fix: change "make sure revert" to "make sure to revert". I forgot to put "to" in the original text. :-D

**Comment by jhinkey** [comment]
It looks good, JR!

**Comment by jrhoun** [comment]
Thanks Jim!

**Pull request closed without merging**